make-rules/shared-macros.mk

```diff
@@ -359,6 +359,8 @@ export PARFAIT_NATIVEGXX=$(GCC_ROOT)/bin/g++
 # is currently the default) to not even define the usage of wrapper in the
 # userland-building makefile system.
 # If you want to speed up your re-builds, you must set ENABLE_CCACHE=true.
+# For legacy reasons, the inverted CCACHE_DISABLE variable (configuration of the
```

Comment:
I'm still not convinced that we need two variants of the var. We have no "leagacy reasons", code is fresh :)